Tests/Twig/Extension/PageExtensionTest.php

```diff
@@ -102,4 +104,13 @@ public function testControllerWithoutSite()
         $HttpKernelExtension->expects($this->once())->method('controller')->with('bar', array(), array());
         $extension->controller('bar');
     }
+
+    private function skipInPHP55()
+    {
+        if (strpos(phpversion(), '5.5') !== false) {
```

Comment:
Since version_compare cant ignore minor version, i created a double if to check if higher than 5.5 and lower than 5.6. Like it this way?